starters/solidjs-tanstack-tailwind/src/components/Tester/index.jsx

```diff
@@ -0,0 +1,21 @@
+import { createSignal } from 'solid-js';
+
+function Tester() {
+  const [todos, setTodos] = createSignal([]);
+
+  const toggleTodo = (value) => {
+    if (todos().includes(value)) {
+      const updateTodo = todos().filter((res) => res !== value);
+      setTodo([...updateTodo]);
+    } else if (todos().length > 0) {
```

Comment:
Love the defensiveness, but it might not be needed here.

```javascript
let todos = [];
const value = 'newItem';
if (todos.length > 0) {
  todos = [...todos, value];
} else {
  todos = [value];
}

// todos = ['newItem']
```

has the same result as

```javascript
let todos = [];
const value = 'newItem';

todos = [...todos, value];

// todos = ['newItem']
```